ee/clickhouse/migrations/0013_persons_distinct_ids_column.py

```diff
@@ -0,0 +1,14 @@
+from infi.clickhouse_orm import migrations
+
+from ee.clickhouse.sql.person import KAFKA_PERSONS_TABLE_SQL, PERSONS_TABLE_MV_SQL
+from posthog.settings import CLICKHOUSE_CLUSTER
+
+operations = [
+    migrations.RunSQL(f"DROP TABLE person_mv ON CLUSTER {CLICKHOUSE_CLUSTER}"),
+    migrations.RunSQL(f"DROP TABLE kafka_person ON CLUSTER {CLICKHOUSE_CLUSTER}"),
+    migrations.RunSQL(
+        f"ALTER TABLE person ON CLUSTER {CLICKHOUSE_CLUSTER} ADD COLUMN IF NOT EXISTS distinct_ids Array(VARCHAR)"
```

Comment:
Q: Does this work without a default? What happens to `distinct_ids` on pre-existing persons?

Reply:
They are not populated. Forgot to mention in the description that if the experiment is successful, we'll need to back-fill these values.